ItalianTranslation_PlanetCoaster.xml

```diff
@@ -103,7 +103,7 @@
 		<entry key="AvatarCreator_CreateAdultGroup" translation="Crea Gruppo di Adulti"/>
 		<entry key="AvatarCreator_CreateFamilyGroup" translation="Crea Gruppo Famiglia"/>
 		<entry key="AvatarCreator_CreateGroup" translation="Crea un Gruppo!"/>
-		<entry key="AvatarCreator_CreateGroupDescription" translation="Aggiungi e Rimuovi Ospiti dal Gruppo. !PLACEHOLDER TEXT!"/>
```

Comment:
Ok senza dubbio chi prima arriva viene sovrascritto (non so se lo hai corretto o è effettivamente una sovrascrittura)

Reply:
Si, ho fatto diverse correzioni. L'originale in inglese era diverso.